setup.py

```diff
@@ -113,6 +113,7 @@
 package_info['package_data'].setdefault(PACKAGENAME, [])
 package_info['package_data'][PACKAGENAME].append('data/*')
 package_info['package_data'][PACKAGENAME].append('io/schemas/*')
+package_info['package_data'][PACKAGENAME].append('io/tests/data/*')
```

Comment:
use this `https://github.com/tardis-sn/tardis/blob/master/tardis/io/setup_package.py` for adding that.